src/citra_qt/configuration/config.cpp

```diff
@@ -14,6 +14,10 @@
 #include "input_common/main.h"
 #include "input_common/udp/client.h"
 #include "network/network.h"
+#ifdef ARCHITECTURE_x86_64
+#include "common/x64/cpu_detect.h"
+#endif
+#include "QSysInfo"
```

Comment:
This is a library header, so prefer `<>` when referring to it:

```suggestion
#include <QSysInfo>
```

Reply:
Done and done